crypto/x509/pcy_data.c

```diff
@@ -55,6 +55,7 @@ X509_POLICY_DATA *policy_data_new(POLICYINFO *policy,
     ret = OPENSSL_zalloc(sizeof(*ret));
     if (ret == NULL) {
         X509V3err(X509V3_F_POLICY_DATA_NEW, ERR_R_MALLOC_FAILURE);
+        ASN1_OBJECT_free(id);
```

Comment:
Small nit - maybe move this line up so it looks similar to the code below it.

Reply:
Updated.